src/pages/visualization-view/visualization-view.js

```diff
@@ -80,11 +88,24 @@ const VisualizationView = (props) => {
             </Tab>
           </span>
           <span className='action-area'>
-            <React.Fragment>
+            <React.Fragment >
+              <TabButton className={`button-${isAuthenticated ? 'enabled' : 'disabled'}`} onClick={showLoginPrompt}>
+                <div title='Saved Visualizations'>
+                  <a href='/user' className={`header-item link-${isAuthenticated ? 'enabled' : 'disabled'}`}>
+                    <img className={`folder-icon`} src={folderIcon} />
+                  </a>
+                </div>
+              </TabButton>
+              <AutoAnchoringPopover className='login-prompt popover-anchor' open={!isAuthenticated && userNeedsLoginInfo} onClose={closeLoginPrompt}>
+                <div className="login-message">
+                <p>You need to be logged in to see your visualizations</p>
+                <Auth0LoginZone/>
+                </div>
+              </AutoAnchoringPopover>
```

Comment:
Did you look into DRYing up this `TabButton/AutoAnchoringPopover`combo platter?  Seeing as how we now have a pattern for saving and viewing visualizations, it seems like it would be nice to combine these into some kind of shared component that could be re-used, but I understand there may be a LOT of properties to have to pass down.  Just wondering if you explored it.

Reply:
I didn't, but I could look into it today